latest/index.bs

```diff
@@ -207,8 +207,52 @@ keys as specified below for discovering certain types of data, especially images
 
 Metadata about the multiple resolution representations of the image can be
 found under the "multiscales" key in the group-level metadata.
-The specification for the multiscale (i.e. "resolution") metadata is provided
-in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+"multiscales" MUST contain the field "datasets", which is a list of dictionaries describing
+the arrays storing the individual resolution levels. Each dictionary in "datasets" MUST
+contain the field "path", whose value contains the path to the array for this resolution relative
+to the root zarr group. The "path"s MUST be ordered from largest (i.e. highest resolution) to smallest.
```

Comment:
I would have thought it's relative to the current zarr group (containing the multiscales) as opposed to the root zarr group.

Reply:
Makes sense. I will change it.